helm-chart/kuberay-apiserver/templates/role.yaml

```diff
@@ -0,0 +1,42 @@
+
+{{- if .Values.rbacEnable }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ .Values.name }}
+  name: {{ .Values.name }}
+rules:
+- apiGroups:
+  - ray.io
+  resources:
+  - rayclusters
```

Comment:
RayServe and RayJob support is not added to ApiServer yet, right? So we just need Rayclusters now

Reply:
Yes, this role only granted to resource rayclusters now.